packages/components/src/utils/hooks/index.js

```diff
@@ -1,3 +1,5 @@
 export { default as useControlledState } from './use-controlled-state';
 export { default as useJumpStep } from './use-jump-step';
 export { default as useUpdateEffect } from './use-update-effect';
+export { default as useControlledValue } from './use-controlled-value';
+export { default as usePropRef } from './use-prop-ref';
```

Comment:
Both of these should go into compose I think unless we want to avoid exposing them as public APIs. We can always make them experimental/unstable if we wanted to, but they probably better fit in `compose`. @gziolo do you have any opinion here?

Reply:
I'd personally leave this type of decision for later. It's perfectly fine to keep those hooks internal and present only in `@wordpress/components` for now. It's better to have a single PR per every new API we want to expose because it makes the discussion focused. Otherwise, you could have this PR opened for months because some parts of the public API might seem controversial, or folks could have objections if it is necessary to introduce some hooks as public API. I vote to finish building the UI  for Global Styles first and introduce new public APIs as we see to use them in other packages, or we know that it is so popular internally that it begs to get exposed to the world.

Reply:
@ramonjd Looks like this is the answer we can follow for the rest of the places I asked the same question.

Thanks Greg!